definitions/palabritas.md

```diff
@@ -326,12 +329,52 @@ For example:
 ### Finishing the game
 `-> END`
 
+### Boomerang divert
+
+Another way to access a section is using the `boomerang divert` command `<->`. Unlike the regular `divert`, the `boomerang divert` command will take you back to your original spot once you're done with the section. 
+
+```cuentitos
+  * I go to bed
+    Feeling depleted of spoons, you go right back to bed.
+    set energy 10
+    set day +1
+    set time -7.5
+    <-> second_day
+  
+    At last the end has come.
+    -> END
+  
+# second_day
+  You wake up feeling refreshed. Let's see what this day brings.
+```
+
+In this example, once the section second_day finishes, the story goes back to where the `boomerang divert` was and shows the line `At last the end has come.`.
+
+### Unique command
+
+To make sure that the story never reaches the same spot twice, you can use the `unique` command.
+
+```cuentitos
```

Comment:
Please, add something that can be included in the example story, and continue that story instead.